components/Datepicker/style.module.scss

```diff
@@ -0,0 +1,58 @@
+@import "styles/index";
+
+.mdl-container-wrapper {
+	
+	:global(.react-datepicker) {
+		box-shadow: 0 1px 2px 0 rgba(0, 0, 0, 0.15);
+		border-radius: 2px;
+		border: none;
+		background-color: $color-m-white;
+		font-family: $font-family;
+		font-size: $base-font-size;
+	}
+
+	:global(.react-datepicker__header) {
```

Comment:
There's lots of opportunity for tightening this whole file up with SASS. E.g., change this to just `&__header {` and move it into the `:global(.react-datepicker) {` block